megamek/src/megamek/common/Tank.java

```diff
@@ -1677,32 +1680,32 @@ public double getPriceMultiplier() {
                   (movementMode.equals(EntityMovementMode.NAVAL) ||
                          movementMode.equals(EntityMovementMode.HYDROFOIL) ||
                          movementMode.equals(EntityMovementMode.SUBMARINE))) {
-            priceMultiplier *= weight / 100000.0;
+            priceMultiplier *= 1 + (weight / 100000.0);
         } else {
             switch (movementMode) {
```

Comment:
could change this switch to a more modern version, but this is of no consequence.